AyuntamientoMadrid/consul
# Change concept of current budget to account for multiple budgets

Where
=====
* **Related PR's:** https://github.com/consul/consul/pull/2322
* **Related Issues:** Closes https://github.com/consul/consul/pull/2322

What
====
- Add concept of current budget
- Fix bug displaying 2016 investments in the budget's welcome page https://github.com/AyuntamientoMadrid/consul/pull/1118/commits/bdfa438e8bc9674302fa1c6e1327d3210f9fd3e0

Test
====
- Manually tested in staging environment 👍 

Deployment
==========
- As usual

Warnings
========
- There would be a problem if there are no budgets available
https://github.com/consul/consul/pull/2322#discussion_r161633830
Solving soon!

- There is a non-relevant Gemfile.lock update, should have fixed it in a separate PR, no biggie :)
